Note which Rust versions are supported

FWIW, I followed the directions for at [Taking Spin for a spin \| Spin Documentation](https://spin.fermyon.dev/quickstart) and got this error:

```
$ spin build
...
   Compiling wit-bindgen-gen-rust v0.2.0 (https://github.com/bytecodealliance/wit-bindgen?rev=cb871cfa1ee460b51eb1d144b175b9aab9c50aba#cb871cfa)
error: there is no argument named `n`
   --> /home/neal/.cargo/git/checkouts/wit-bindgen-f98ccff846621481/cb871cf/crates/gen-rust/src/lib.rs:480:39
    |
480 |                         write!(name, "{n}").unwrap();
    |                                       ^^^

error: there is no argument named `name`
   --> /home/neal/.cargo/git/checkouts/wit-bindgen-f98ccff846621481/cb871cf/crates/gen-rust/src/lib.rs:621:46
    |
621 |             self.push_str(&format!("pub enum {name}"));
    |                                              ^^^^^^

error[E0658]: use of unstable library feature 'iter_zip'
 --> /home/neal/.cargo/git/checkouts/wit-bindgen-f98ccff846621481/cb871cf/crates/gen-rust/src/lib.rs:4:5
  |
4 | use std::iter::zip;
  |     ^^^^^^^^^^^^^^
  |
  = note: see issue #83574 <https://github.com/rust-lang/rust/issues/83574> for more information

error[E0658]: use of unstable library feature 'iter_zip'
   --> /home/neal/.cargo/git/checkouts/wit-bindgen-f98ccff846621481/cb871cf/crates/gen-rust/src/lib.rs:595:13
    |
595 |             zip(self.union_case_names(iface, union), &union.cases)
    |             ^^^
    |
    = note: see issue #83574 <https://github.com/rust-lang/rust/issues/83574> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `wit-bindgen-gen-rust` due to 4 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
Error: Build command for component spin-hello-world failed with status Exited(101).

```

I checked for a Rust update:
```
$ rustup check
stable-x86_64-unknown-linux-gnu - Update available : 1.57.0 (f1edd0429 2021-11-29) -> 1.64.0 (a55dd71d5 2022-09-19)
rustup - Update available : 1.24.3 -> 1.25.1
```

and did a `rustup update` to upgrade. Then the build went smoothly, and it ran like a champ. [Cool!]

So I suggest adding a note to the quickstart documenting a working version of the Rust toolchain, or at least suggesting a check.

--- a/docs/content/quickstart.md
+++ b/docs/content/quickstart.md
@@ -196,7 +196,6 @@ Successfully ran the build command for the Spin components.
 > For more details, see the [page about developing Spin applications](./developing.md).
 
 If you run into errors, you might want to use `rustup check` to see if your Rust installation is up-to-date.
-This quickstart is known to work with Rust version 1.64.
 
 ## Running the application with `spin up`
 

--- a/docs/content/quickstart.md
+++ b/docs/content/quickstart.md
@@ -195,6 +195,9 @@ Successfully ran the build command for the Spin components.
 >
 > For more details, see the [page about developing Spin applications](./developing.md).
 
+If you run into errors, you might want to use `rustup check` to see if your Rust installation is up-to-date.
+This quickstart is known to work with Rust version 1.64.
+
 ## Running the application with `spin up`
 
 Now that we configured the application and built our component, we can _spin up_

--- a/docs/content/rust-components.md
+++ b/docs/content/rust-components.md
@@ -281,8 +281,8 @@ be used when implementing the component.
 Sometimes things can go wrong, especially such early projects. If you bump into
 issues building and running your Rust component:
 
-- ensure `cargo` is present in your path — we recommend
-[Rust](https://www.rust-lang.org/) at [1.56+](https://www.rust-lang.org/tools/install)
+- ensure `cargo` is present in your path
+- ensure your [Rust](https://www.rust-lang.org/) version is recent, at [1.56+](https://www.rust-lang.org/tools/install)
 - ensure `wasm32-wasi` target is configured for your Rust installation —
 you can add it by running `rustup target add wasm32-wasi`
 - build a `release` version of the component — all Spin application definitions

--- a/docs/content/rust-components.md
+++ b/docs/content/rust-components.md
@@ -282,7 +282,7 @@ Sometimes things can go wrong, especially such early projects. If you bump into
 issues building and running your Rust component:
 
 - ensure `cargo` is present in your path
-- ensure your [Rust](https://www.rust-lang.org/) version is recent, at [1.56+](https://www.rust-lang.org/tools/install)
+- ensure your [Rust](https://www.rust-lang.org/) version is recent, via `rustup check`
 - ensure `wasm32-wasi` target is configured for your Rust installation —
 you can add it by running `rustup target add wasm32-wasi`
 - build a `release` version of the component — all Spin application definitions

